### ISSUE TITLE
16 - New detail panel for the Asset Browser inside inspector

### ISSUE THREAD

**Issue by lsemp3d** [o3de/o3de#12856]
**Use Story:**

As a user I need a detail view panel that will replace the inspector details when user is view content inside AB. The current details in the AB panel should be moved over to the inspector panel and removed from the AB. Currently the Detail panel is in spot where it cannot grow beyond the small space taken up in the asset browser right panel.

O3DE currently

![image](https://user-images.githubusercontent.com/58790905/198411280-fdb0bb07-0d51-4827-8dc6-bf7e12b5faee.png)

 **Notes:**

Some of the content that we should add is: date modified, merged date, who changed it last, associated assets, file size, etc. This should appear with the asset details in the right hand panel.

 

 

**Previous ticket details:**

The current asset preview viewport in the Asset Browser/Asset Picker has the following issues-
 - As a user, I want the metadata have more prominance and space to read the details. Right now the thumbnail preview for the cube map is tall and verticle, and pushes it down drastically.
 - The cube map preview is shaped vertically, which makes it hard to view the details of the cube map, and there's no way to zoom in the image either.

**Comment by bhanuja-s** [comment]
After speaking with @rainbj, we decided this ticker requires some background context. First, this ticket relates to https://github.com/o3de/o3de/issues/12872 and https://github.com/o3de/o3de/issues/12876. 

The ideal UX solution is a contextual Inspector panel, but after @rainbj's earlier discussions with @lsemp3d—we decided to create phases to the Inspector panel work. We begin by migrating the details panel into an Inspector panel, creating a separate Asset Browser Inspector, which will then lead into the critical work of creating a context-aware Inspector panel.

I hope this provides further information, feel free to reach out for more behaviour-related questions (if any). Thanks!
